odltools/karaf/cli.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2018 Red Hat, Inc. and others. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from odltools.karaf import dump
+
+
+def add_dump_parser(parsers):
```

Comment:
The name of the parameter is dump so that is what I named the method. I think that what you suggest makes sense if there is no -p option and the command always pretty prints.

Reply:
But it's not a parser of a log file, what is being added is a parser for the command line args for the dump command.

Reply:
yeah I mean what is the plan for this karaf module?  You mentioned you are going to add more things.  Are they going to be part of the same overall parser?  I would just make a single parser like you have for 'karaf', and hten add arguments to execute each option (like pretty printing).